### ISSUE TITLE
All categories say no covers available

### ISSUE THREAD

**Issue by langerak** [nZEDb/nZEDb#2248]
Hi,

I'm currently running the 448 release:

```
root@indexer:/var/www/nZEDb# ./zed update nzedb
Already up-to-date.

Running composer install process...
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
Checking database version...
Db: 448,        File: 448
Up to date.
The Smarty compiled template cache has been cleaned for you
```

Everything works fine, as mentioned by the developer a couple of days ago, the Python scripts are not developed anymore and started using the PHP equivalents.

I currently have around 200000 releases active in my installation and I see alot of NZB's and covers in the resources folder. One of the things that I cannot seem to get working is the cover view for the following categories:
- XXX
- Movies
- Games
- Music
- Ebooks
- Console

It always says that it's either disabled (which it is not) or my keys are wrong. The last one is what I've checked by creating my own API keys for all relevant sites which is reported working with the API test scripts.

In the standard logging with PHP and MySQL I cannot determine what causes this issue and what steps I can perform in order to solve it. Permissions for the resources folder are set to 0777 and is a simple Apache, PHP and MySQL installation on Ubuntu 16.04.

Since my installation itself is working rather fine I don't want to start all over again. Does anybody know of this issue and how it can be solved?

If more information is needed, let me know!

**Comment by DariusIII** [comment]
When did these problems start? After the update?

**Comment by langerak** [comment]
Hi,

Good question! I was not clear I see. It's the version I'm running since the start (installation is only 12 days old). To be honest: from the start with this installation it has never worked. Strangely enough it does create covers and such.

**Comment by DariusIII** [comment]
Check apache error logs (/var/log/apache2/error.log) and also enable debugging in settings.php (all of the options need to be set to true (notices, infos, warning, error, fatal).
Edit: can you paste your apache/nginx nZEDb.conf?

**Comment by langerak** [comment]
I've enabled debugging and when I click on the Movies category for example, on the top of the page the following error is shown:

`SQLSTATE[42000]: Syntax error or access violation: 1055 Expression #34 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'nzedb.g.name' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by`

I'm running MySQL 5.7.

**Comment by DariusIII** [comment]
Ok. I have never seen this error on nZEDb, i know nn+ has some problems related to this. Have you maybe added sql_mode=only_full_group_by to your my.cnf?

**Comment by langerak** [comment]
I've checked it but it is not forced by the configuration:

```
root@indexer:/etc/mysql# grep -R sql_mode ./
root@indexer:/etc/mysql#
```

When Googling the issue alot of people have this issue with other software as well, so it seems that there were some big changes regarding the accepted SQL modes in the transition from 5.6 to 5.7.

**Comment by DariusIII** [comment]
What do you use mysql, percona or mariadb?
Edit: Upgrade to mariadb anyway.
https://downloads.mariadb.org/mariadb/repositories/#mirror=netinch

**Comment by langerak** [comment]
I currently use MySQL 5.7:

```
root@indexer:/etc/mysql# mysql --version
mysql  Ver 14.14 Distrib 5.7.13, for Linux (x86_64) using  EditLine wrapper
```

**Comment by DariusIII** [comment]
Upgrade to mariadb 10.1 it will work wonders for you.

**Comment by Bart39** [comment]
http://stackoverflow.com/a/37248560

**Comment by langerak** [comment]
I've made a SQL dump of the nzedb database and purged all mysql packages that were installed. I then added the MariaDB 10.1 repo for Ubuntu 16.04 and installed all MariaDB related packages on the system.

After importing the database and changing the config from socket to port it all works again! Thanks!

So it seems that nZEDb won't work flawless on Ubuntu 16.04 afterwards because of MySQL 5.7, but at least there is a workaround. Good for documentation as well!

Again, thanks alot!
